client/src/unifyfs-internal.h

```diff
@@ -266,26 +266,27 @@ typedef struct {
 } unifyfs_chunkmeta_t;
 
 typedef struct {
-    off_t global_size;               /* Global size of the file */
-    off_t local_size;                /* Local size of the file */
-    off_t log_size;                  /* Log size.  This is the sum of all the
-                                      * write counts. */
-    pthread_spinlock_t fspinlock;    /* file lock variable */
-    enum flock_enum flock_status;    /* file lock status */
-
-    int storage;                     /* FILE_STORAGE type */
-
-    int gfid;                        /* global file id for this file */
-    int needs_sync;                  /* have unsynced writes */
-
-    off_t chunks;                   /* number of chunks allocated to file */
-    off_t chunkmeta_idx;            /* starting index in unifyfs_chunkmeta */
-    int is_laminated;               /* Is this file laminated */
-    uint32_t mode;                  /* st_mode bits.  This has file
-                                     * permission info and will tell you if this
-                                     * is a regular file or directory. */
-    struct seg_tree seg_tree;       /* Segment tree containing our coalesced
-                                     * writes */
+    off_t global_size;            /* Global size of the file */
+    off_t local_size;             /* Local size of the file */
+    off_t log_size;               /* Log size.  This is the sum of all the
+                                   * write counts. */
+    pthread_spinlock_t fspinlock; /* file lock variable */
+    enum flock_enum flock_status; /* file lock status */
+
+    int storage;                  /* FILE_STORAGE type */
+
+    int gfid;                     /* global file id for this file */
+    int needs_sync;               /* have unsynced writes */
+
+    off_t chunks;                 /* number of chunks allocated to file */
+    off_t chunkmeta_idx;          /* starting index in unifyfs_chunkmeta */
+    int is_laminated;             /* Is this file laminated */
+    uint32_t mode;                /* st_mode bits.  This has file
+                                   * permission info and will tell you if this
+                                   * is a regular file or directory. */
+    struct seg_tree seg_tree;     /* Segment tree containing our coalesced
+                                   * writes */
+    struct seg_tree extents;      /* Segment log of all local data */
```

Comment:
We should rename these, since they're both seg_trees and both store extents.  Maybe:
```
struct seg_tree write_tree;     /* Segment tree of the writes since last sync */
struct seg_tree all_write_tree; /* Segment tree of all the writes we have ever done */
```

Reply:
Went with "extents_sync" and "extents" here.  Sound good?

Reply:
Yea, I like that even better